src/__fixtures__/registry.ts

```diff
@@ -1,6 +1,7 @@
-import { ChildProcess, spawn } from 'child_process';
+import execa from 'execa';
```

Comment:
what's this? I don't see it in package.json. maybe I'm blind.

Reply:
It's in the dependencies

Reply:
oops yep i see it.